functorch/test/test_control_flow.py

```diff
@@ -167,7 +171,7 @@ def f(x, y):
         with self.assertRaises(AssertionError):
             make_fx(f)(x, torch.tensor(False))
 
-
+    @unittest.expectedFailure
```

Comment:
Not real failure anymore cause we have symbolic shapes, can delete as shape should not be in meta.

Reply:
Are you suggesting we should delete the test? I'm not sure what the behavior of cond should be when symbolic shapes are involved. At the very least I'd expect some guards being added that the symbolic shapes traced through one of the paths is the same as the shapes traced through the other past